browser/src/Editor/OniEditor/OniEditor.tsx

```diff
@@ -170,6 +171,10 @@ export class OniEditor implements IEditor {
             buf => extensions.includes(path.extname(buf.filePath)),
             buf => new ImageBufferLayer(buf),
         )
+
+        windowManager.onUnhandledMove.subscribe((direction: string) => {
```

Comment:
Just saw your comment - there is one problem with the location here. In the 'multiplexing' scenario (when `editor.split.mode` is `'oni'`), we will potentially have _multiple_ `OniEditor` instances - so that means this would get fired multiple times.

One option would be to plop this code in `browser/src/index.tsx`, after we've created the `WindowManager`. Or add an `activate` method in `MultiProcess`, like we have for some of the other services, pass in the `windowManager` as a dependency, and wire up the event there. Let me know if that isn't clear